.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
This file specify very simple, but really needed rules, like:
 - indent using tabs or spaces
 - indent size
 - charset
 - end of line style
 - should editor insert final newline
 - should editor removes whitespaces